CHANGELOG.md

```diff
@@ -119,7 +130,7 @@
 - [dev] Turns on "keychain sharing" to fix a keychain bug in sim - orta
 - GeneVC now shows about information, and trending artists - orta
 - GeneVC now shows artworks for the gene - orta
-- Show ‘bid now’ call to action on artworks that are in an auction - alloy
+- Show "bid now" call to action on artworks that are in an auction - alloy
```

Comment:
So that I could add the spell checker as a recommended extension for emission, it can't handle smart-quotes :D